docs/source/cuda_environment_variables.rst

```diff
@@ -29,6 +29,8 @@ For more information on CUDA runtime environment variables, see `CUDA Environmen
     - If set to ``0``, enables fallback to record streams-based synchronization behavior in NCCL.
   * - ``TORCH_CUDNN_V8_API_DEBUG``
     - If set to ``1``, sanity check whether cuDNN V8 is being used.
+  * - ``TORCH_TRITON_SKIP_CC_CHECKS``
```

Comment:
Thanks for looking into documenting this - however, this doc is torch only envvars. we have a bunch of torchinductor envvars that are just in torch/_inductor/config. delete ? not sure this belongs here